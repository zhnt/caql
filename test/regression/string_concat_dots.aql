// Test string concatenation using + operator (simplified)
print("hello" + "world");
print("simple" + "test");

let s1 = "foo";
let s2 = "bar";
print(s1 + s2);

let result = "test" + "123";
print(result);

// Mixed with variables
let prefix = "Hello, ";
let name = "AQL";
let greeting = prefix + name + "!";
print(greeting);
