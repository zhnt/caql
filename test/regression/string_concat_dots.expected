helloworld
simpletest
foobar
test123
Hello, AQL!
